geoips/geoips_utils.py

```diff
@@ -36,8 +36,10 @@ def get_entry_point_group(group):
     if sys.version_info[:3] >= (3, 10, 0):
         eps = metadata.entry_points(group=group)
     else:
-        eps = metadata.entry_points()[group]
-
+        raise NotSupportedError(
+            "GeoIPS only supports python versions >= 3.10.0. Please upgrade your "
+            "Python Version to use GeoIPS."
+        )
```

Comment:
I'm debating whether this is required. GeoIPS can't even be installed with a lower version of Python at this point. 

In fact, is this function still required or should we just call `metadata.entry_points(group=group)` anywhere this is needed?

Reply:
I'm going to keep the function there as it's used in other places but I'll remove the support for Python versions <= 3.9.

Reply:
I think that, since it is only used in two locations and it is just a one-liner, I'd prefer to remove the function and directly call metadata.entry_points(group=group). It seems like overkill to have a function defined for this.

Reply:
That's fine with me! It ended up being used in more spots than I thought but I removed all of those instances and replaced it with ``metadata.entrypoints(group=group)``.

Reply:
Thanks @evrose54